Of course, the code before the fix fails the newly added tests.

```shell
$ go test ./pixel/
--- FAIL: TestImageRGB555 (0.00s)
    image_test.go:70: failed to roundtrip color: expected {255 0 0 255} but got {0 0 255 255}
    image_test.go:70: failed to roundtrip color: expected {0 0 255 255} but got {255 0 0 255}
    image_test.go:70: failed to roundtrip color: expected {16 0 0 255} but got {0 0 16 255}
    image_test.go:70: failed to roundtrip color: expected {0 0 16 255} but got {16 0 0 255}
FAIL
FAIL    tinygo.org/x/drivers/pixel      1.456s
FAIL
```

--- a/pixel/image_test.go
+++ b/pixel/image_test.go
@@ -109,7 +109,7 @@ func TestImageRGB444BE(t *testing.T) {
 
 func TestImageMonochrome(t *testing.T) {
 	image := pixel.NewImage[pixel.Monochrome](128, 64)
-	if width, height := image.Size(); width != 128 && height != 64 {
+	if width, height := image.Size(); width != 128 || height != 64 {
 		t.Errorf("image.Size(): expected 128, 64 but got %d, %d", width, height)
 	}
 	for _, expected := range []color.RGBA{

